Sources/PointFree/Atom.swift

```diff
@@ -34,8 +34,15 @@ public let atomLayout = View<AtomFeed> { atomFeed -> [Node] in
       [xmlns("http://www.w3.org/2005/Atom")],
       [
         title(atomFeed.title),
-        link([href(atomFeed.atomUrl), rel(.self)]),
-        link([href(atomFeed.siteUrl)]),
+        element(
+          "link",
+          [
+            .init("href", atomFeed.atomUrl) as Attribute<Void>,
+            .init("rel", "self")
```

Comment:
I can't use any of the attribute helpers because their `HasABC` restrictions don't match up for what I need. I also need to cast to `Attribute<Void>` to take care of the phantom type.